### DISCUSSION TITLE
Hello is there a setting in Open-Shell that allows for the tracking of recently installed programs (not recently used items)?

### DISCUSSION THREAD

**Discussion by kris3713** [Open-Shell/Open-Shell-Menu#1545] [Q&A] [answered]
Literally what the title says ⬆.

**Comment by ge0rdi** [comment]
`General Behavior` -> `Highlight newly installed programs` ?

**Comment by kris3713** [comment]
@ge0rdi I want the newly installed program to appear in the **recent programs** part of the menu, do I have to download a pre-release version for this to happen?
![(firefox)A7J4p4_05-16-2023](https://github.com/Open-Shell/Open-Shell-Menu/assets/101912712/17023374-2acb-40e7-922d-05b24713f22d)

**Comment by ge0rdi** [comment, marked as answer]
`Recent Programs` contains _recently used_ programs.
You need to actually run those installed programs. Then they should appear there.

**Comment by kris3713** [comment]
Alright, I'll see if that works for me.
